ASAManager/run.sh

```diff
@@ -16,9 +16,9 @@ start_containers() {
     cd $APP_HOME
     ./scripts/storageadapter.sh start
     ./scripts/telemetry.sh start
+    ./scripts/devicesimulation.sh start
```

Comment:
why this change?

Reply:
pcs config depends on storage adapter, telemetry and device simulation and hence this change